File: Backend/Service/Models/Client/ClientModel.cs
using Service.Models.Classifier;

namespace Service.Models.Client
{
    public class ClientModel
    {
            public int Id { get; set; }
            public string FirstName { get; set; }
            public string LastName {get; set; }
            public string Phone {get; set; }
            public string Nit {get; set; }
            public byte[] File {get; set; }
            public string Photo {get; set; }
            public int? ClientZoneId { get; set; }
            public string Ubication {get; set; }
            public string Company {get; set; }
            public bool? HasPhoto {get; set; }
            public int? ClientRateId { get; set; }
            public int? ClientTypeId { get; set; }
            public int? ClientDiscountId { get; set; }
            public bool? IsActive {get; set; }
            public BaseClassifierModel Zone { get; set; }
    }
}

File: App/src/app/modules/clients/models/client_model.ts
export interface Client {
  id: number;
  firstName: string;
  lastName: string;
  phone: string | null;
  ci: string | null;
  nit: string | null;
  file: string | null;
  photo: string | null;
  clientZoneId: number | null;
  clientRateId: number | null;
  clientTypeId: number | null;
  clientDiscountId: number | null;
  ubication: string | null;
  company: string | null;
  latitude: number | null;
  longitude: number | null;
  reference: string | null;
  link: string | null;
  hasPhoto: boolean;
  isActive: boolean;
  zone: {
    id: number;
    name: string;
    description: string;
    isActive: boolean;
  };
}